contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -48,6 +48,7 @@ stages:
             workingDirectory: "$(ONEFUZZ_DEPLOY_LOC)"
             script: |
               set -ex
+              python -m pip install -U wheel
```

Comment:
I think this one may be unnecessary, because it is definitely insufficient. What really matters is the `wheel` version used in the `tox` environment (where `pipenv check` gets invoked). Let me try removing it.